packages/mgt-components/src/components/mgt-people-picker/mgt-people-picker.ts

```diff
@@ -968,17 +1040,23 @@ export class MgtPeoplePicker extends MgtTemplatedComponent {
             } else {
               const isUserOrContactType = this.userType === UserType.user || this.userType === UserType.contact;
               if (this._userFilters && isUserOrContactType) {
-                people = await getUsers(graph, this._userFilters, this.showMax);
               } else {
                 people = await getPeople(graph, this.userType, this._peopleFilters);
               }
             }
+            people = await getUsers(graph, this._userFilters, this.showMax);
```

Comment:
It seems like this change broke the `user-ids` property. 

The component will always load the top x users, even if you specify user ids, since it overwrites the `people` array that is filled from the user ids query.

Reply:
Thanks for spotting this @AndersSahlin. 

Looks to me like this needs to be moved back to where it was inside the if case to remove this regression.